integration-cli/docker_cli_pull_test.go

```diff
@@ -153,9 +149,7 @@ func (s *DockerHubPullSuite) TestPullClientDisconnect(c *check.C) {
 		if _, err := s.CmdWithError("inspect", repoName); err == nil {
 			break
 		}
-		if i >= maxAttempts {
-			c.Fatal("timeout reached: image was not pulled after client disconnected")
-		}
+		c.Assert(i >= maxAttempts, checker.Equals, false, check.Commentf("timeout reached: image was not pulled after client disconnected"))
```

Comment:
same.
this commentf can be a standard comment on the same line as there are no variables being interpolated.